app/service/file_svc.py

```diff
@@ -70,9 +70,10 @@ def __init__(self, plugins, exfil_dir):
         :return: a tuple: the plugin the file is found in & the relative file path
         """
         for plugin in self.plugins:
-            file_path = await self._walk_file_path('plugins/%s/%s' % (plugin, location), name)
-            if file_path:
-                return plugin, file_path
+            for subd in ['', 'data/']:
+                file_path = await self._walk_file_path('plugins/%s/%s%s' % (plugin, subd, location), name)
```

Comment:
remove slash from subdirectory name and I'd recommend using pathlib or sys to concatenate paths